uber/site_sections/accounts.py

```diff
@@ -115,7 +115,7 @@ def reset(self, session, message='', email=None):
         }
 
     @unrestricted
-    def change_password(self, session, message='', old_password=None, new_password=None, csrf_token=None):
+    def change_password(self, session, message='', old_password=None, new_password=None, csrf_token=None, confirm_new_password=None):
```

Comment:
+1

Minor note for next time: there's no reason to include this as an HTTP parameter, since it's never actually used on the backend.  If you create a form element without a ``name`` attribute, then the browser won't submit it to the backend.  So if we want to implement something like a confirm password field, we could give it an ``id`` but no ``name`` and then it doesn't get submitted and wouldn't need to be defined here.